### PR TITLE
fix syslink typo

### PR DESCRIPTION
**Pull request by asklar**

### CODE REVIEW

**Comment by PRMerger18** [comment]
@asklar : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: desktop-src/Controls/use-syslink-notifications.md [hunk 1]
```diff
@@ -1,80 +1,81 @@
...
+[Using SysLink Controls](using-syslink-controls.md)
+</dt> <dt>
+
+[Windows common controls demo (CppWindowsCommonControls)](https://github.com/microsoftarchive/msdn-code-gallery-microsoft/tree/master/OneCodeTeam/Windows%20common%20controls%20demo%20(CppWindowsCommonControls)/%5BC++%5D-Windows%20common%20controls%20demo%20(CppWindowsCommonControls)/C++/CppWindowsCommonControls)
+</dt> </dl>
+
+ 
+
+ 
+
```
Comment (asklar) [inline desktop-src/Controls/use-syslink-notifications.md:77]:
```suggestion
```

**Comment by PRMerger5** [comment]
@asklar : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: desktop-src/Controls/use-syslink-notifications.md [hunk 2]
```diff
@@ -1,80 +1,80 @@
...
+
+
+diff --git a/desktop-src/Controls/use-syslihnk-notifications.md b/desktop-src/Controls/use-syslihnk-notifications.md
```
Comment (asklar) [inline desktop-src/Controls/use-syslink-notifications.md]:
```suggestion
```

**Comment by PRMerger18** [comment]
@asklar : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by jwmsft — APPROVED** [review]

**Pull request merged**